docs/tasks/logs.rst

```diff
@@ -0,0 +1,259 @@
+Segregate Controller And Datapath Logs
+--------------------------------------
+
+Kanister uses structured logging to ensure that its logs can be easily
+categorized, indexed and searched by downstream log aggregation software.
+
+By default, Kanister logs are output to the controller's ``stderr`` in JSON
+format. Generally, these logs can be categorized into *system logs* and
+*datapath logs*.
+
+System logs are logs emitted by the Kanister to track important controller
+events like interactions with the Kubernetes APIs, CRUD operations on
+blueprints and actionsets etc.
+
+Datapath logs, on the other hand, are logs emitted by data mover task pods
```

Comment:
I am not sure if we can call them data mover task pods. In a lot of scenarios, the pods actually doesn't move data but just runs some command. Example is `KubeExec` function.

Reply:
Also, do we have some other options than data path logs? for example `functionPodLogs` or `kanFunPodLogs` etc.

Reply:
I don't want to leak implementation concepts like `function`, `kanFunc` etc. to the logs. Fundamentally, I want to convey high-level concepts like "system vs. user", "control plane vs data plane" etc. Other than operations like scale up and down, even most `KubeExec` and `KubeOps` calls made by the users involve some kind of data operations, right?

But yes, maybe we can replace "data mover task pods" with just "task pods"?

Reply:
I am more inclined towards changing the value for const `LogKindDatapath` and the reason is, when users see the logs with this extra field they should be able to figure out what are these logs.
I think, `dataPath` is not intuitive enough to tell that these logs are from the commands that are there in the kanister functions. What do you think?

Reply:
What kind of information would you like to convey to the user in the logs? Will using the `ActionSet` and `LogKind` fields together provide the level of granularity you are referring to? Adding more fields to the logs may not necessary make things more intuitive as they tend to clutter the log outputs.

Reply:
I agree, I was not requesting to add more fields. I was asking to change the value `datapath` to something else, if possible. Because, according to me, `datapath` is not able to convey which logs are these exactly.
If possible, we should be able to convey that, these logs are output of the commands that you (users) have  mentioned in the blueprint.

Reply:
do we not want to do anything about it? @pavannd1 @ihcsim 
If not, PR looks good to me.